docs/cli.rst

```diff
@@ -1,11 +1,271 @@
-The ``cci`` command line
+The ``cci`` Command Line
 ========================
 
-Basic operation
+
+
+Basic Operation
 ---------------
 
-Getting help
-------------
+    .. note:: If you are new to working with command line interfaces, the `Install Visual Studio Code <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup/review-base-requirements-install-visual-studio-code?trail_id=build-applications-with-cumulusci>`_ module covers installing and opening a terminal window in Visual Studio Code.
+
+After installing CumulusCI, use the ``cci`` command in your terminal or command prompt to interact with it.
+
+On a macOS, access the terminal via Terminal.app. On Windows, open cmd.exe.
+
+To see all available commands, type ``cci`` in your terminal:
+
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To get information on a specific command listed above, type ``cci <command>``. 
+
+    For example, if you would like more information on the ``task`` command, type ``cci task``:
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+There are even more subcommands available under ``cci task``.
+
+
+
+Getting ``--help``
+------------------
+
+If you want help running a task, type ``cci task run --help``. If you're not certain about what a specific command does, use the ``--help`` option to get more information. 
+
+.. code-block:: console
+
+    $ cci task run --help
```

Comment:
I believe wording/format in terminal has changed drastically. (Or at least it looks that way to me in Windows.) Please confirm.

Reply:
@davisagli Sorry, I commented on the wrong command. This is the one that looks different to me.

Reply:
@justinvholt Given the paragraph above, I think this is intended to show an example of getting help for running a specific task, not for the `cci task run` command as a whole. Which means we need <task_name> in the command. For example run `cci task run deploy --help` to see the options for the `deploy` task.

Or did I misunderstand the intent, @Br4nd0R? We probably should also consider making `cci task run <task_name> --help` print the same thing as `cci task info <task_name>` which also includes the task-specific docs and example usage, not just a list of options.

Reply:
The output for this command (`cci task run --help`) has changed since we did the work for supporting more normalized task option syntax. The output now lists all tasks available to users (same output as `cci task list`) with a brief usage statement of how to run a task at the bottom of the output.

@justinvholt - We should probably change this subheader to something like "Help Running Tasks" and suggest that users run `cci task info <task_name>`. It's slightly annoying that the output for `cci task info <task_name>` and `cci task run <task_name> --help` are formatted differently, but contain (almost) identical information.

We have control over the output in `cci task info <task_name>` which is why we should recommend it over the other way.

Reply:
@Br4nd0R I think we can probably also figure out how to override the output from `cci task run <task_name> --help` if we want to make it consistent.

Reply:
I re-worked this section. Hopefully for the better. I created work items for anything that needs to be corrected on the tooling side.